cassiopeia/src/main/java/edu/asu/diging/gilesecosystem/cassiopeia/web/EditPropertiesController.java

```diff
@@ -36,12 +39,49 @@
     private ISystemMessageHandler messageHandler;
 
     Map<String, String> ocrTypeMap = new HashMap<>();
+    Map<String, String> langTypeMap = new HashMap<>();
+    String defaultLang = new String();
```

Comment:
those should be private (ocrTypeMap as well actually)

Reply:
Made  ocrTypeMap and langTypeMap private Maps